tez-api/src/main/java/org/apache/tez/dag/api/TezConfiguration.java

```diff
@@ -883,6 +883,22 @@ public TezConfiguration(boolean loadDefaults) {
       + "dag.cleanup.on.completion";
   public static final boolean TEZ_AM_DAG_CLEANUP_ON_COMPLETION_DEFAULT = false;
 
+  /**
+   * Integer value. Instructs AM to delete vertex shuffle data if a vertex and all its
+   * child vertices at a certain depth are completed. Value less than 0 indicates the feature
```

Comment:
nit: less than 0...actually 0 also disables the feature

Reply:
ack. Will resolve in next revision.